src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,35 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.10.0
+5 July 2023
+
+### Features
+
+- Now you can select an expiration period when creating a new Personal Access Token. /* 6690 */
```

Comment:
```suggestion
- Now you can select an expiration period when creating a new Personal Access Token.
```

Should we remove all these comments?

Reply:
I would remove them. They don't provide values for customers reading the release notes and customers cannot access to them (not like in the open source product). Internally, we have ways of getting that information again without complexity

Reply:
Oh, ok, my comment was about making it visible in the release notes.

I like that idea 👏